components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -339,11 +339,19 @@ RewardsServiceImpl::~RewardsServiceImpl() {
 }
 
 void RewardsServiceImpl::ConnectionClosed() {
+  if (!ready_->is_signaled()) {
+    BLOG(0, "Ledger process disconnected during startup");
+    return;
+  }
+
+  Reset();
+  BLOG(0, "Restaring ledger process after disconnect");
```

Comment:
Nit: minor typo here (should be "Restarting")

Reply:
Ack, too much coffee today - thanks!